my_list.rb

```diff
@@ -0,0 +1,12 @@
+# my_list.rb
+class MyList
+  include MyEnumerable
```

Comment:
- [ ] Great job adding the `MyList` class 👏. However, please ensure that the `MyEnumerable` class is required. Otherwise, below is the error we get when trying to run the code. :ok_hand:  😊

### Error: :point_down: 
![Screenshot from 2023-10-20 23-30-31](https://github.com/CobbyElsonfx/enumerable-project/assets/89228743/1691a817-fe9d-442f-ad4e-e51cc8ec1661)

#### How to require the `MyEnumerable` class :point_down: 

```
require_relative 'my_enumerable'
```